services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> QuerySet:
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    elif genres_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
+    return Movie.objects.all()
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(pk=movie_id)
```

Comment:
The `get_movie_by_id` function uses `get()`, which will raise a `DoesNotExist` exception if the movie with the given ID is not found. Consider using `get_object_or_404()` if this function is used in a view, or handle the exception appropriately to prevent the application from crashing.